content/scaling-kubernetes-cluster/optimize-database-indexes.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Optimize database indexes"
+description: "Rules and best practices for optimizing database queries and indexes"
```

Comment:
I'd prefer that we be more opinionated. E.g. Optimizing Postgres. Optimizing MySQL. etc.

Reply:
These rules are generic.
They are good for any database.
If we want to describe the steps for a specific database, we probably should put them in different docs.